From f71cc716f05a41e411aa0306d761fe71980f9e52 Mon Sep 17 00:00:00 2001
From: Aelin <aelin@postmarketos.org>
Date: Sat, 12 Sep 2026 14:23:36 +0200
Subject: [PATCH 08/10] Allow systemd-coredump to create
 /run/systemd/coredumpd/kernel

It was previously allowed to bind there, but not create the socket.

I believe this was overlooked in https://github.com/fedora-selinux/selinux-policy/pull/3397
---
 policy/modules/system/systemd.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index 5e97bc2fe..7350a3c8c 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1376,6 +1376,7 @@ mmap_files_pattern(systemd_coredump_t, systemd_coredump_var_lib_t, systemd_cored
 init_var_lib_filetrans(systemd_coredump_t, systemd_coredump_var_lib_t, dir, "coredump")
 
 # /run/systemd/coredumpd/kernel socket
+allow systemd_coredump_t systemd_coredump_var_run_t:dir { add_name write };
 allow systemd_coredump_t systemd_coredump_var_run_t:sock_file manage_sock_file_perms;
 
 kernel_rw_usermodehelper_state(systemd_coredump_t)
-- 
2.55.0

